.gitignore

```diff
@@ -114,3 +114,5 @@ env/
 
 ### Configuration files ###
 .env
+
+config.py
```

Comment:
Do we actually want to have git ignore this, or do we want to delete it?

Reply:
We don't need it, removed.